amplipi/streams.py

```diff
@@ -42,6 +41,7 @@
 
 from amplipi import models
 from amplipi import utils
+from amplipi import app
```

Comment:
pretty sure this re-introduces the bug fixed in https://github.com/micro-nova/AmpliPi/pull/751 ; can you try to run `./scripts/create_spec` and if it doesn't work rearrange your imports?